lib/resources/SignatureRequest.js

```diff
@@ -100,6 +100,11 @@ module.exports = HelloSignResource.extend({
         method: 'POST'
     }),
 
+    embeddedBulkSendWithTemplate: hellosignMethod({
+        path: '/bulk_create_embedded_with_template',
```

Comment:
Is there a specific reason why we're not naming this method `bulkCreateEmbeddedWithTemplate()`? From what I can tell, all other methods generally take the same name as their endpoint, but this one seems to break the mold a bit. Curious if there was a specific reason in mind for this

Reply:
Great call out. Updated to reflect proper name convention.